SnowplowTracker/SnowplowTracker/Emitters/AsyncEmitter.cs

```diff
@@ -161,9 +161,17 @@ public class AsyncEmitter : AbstractEmitter {
 					Log.Debug ("Emitter: EmitLoop shutting down...");
 					break;
 				}
+				List<EventRow> events = new List<EventRow>();
 
+				if (emitLock != null)
+				{
```

Comment:
likewise if you don't mind changing the brace style here too

Reply:
Fixed.